code/model/modelread.cpp

```diff
@@ -4972,9 +4970,70 @@ void swap_bsp_tmappoly( polymodel * pm, ubyte * p )
 	}
 }
 
+void swap_bsp_tmap2poly(polymodel* pm, ubyte* p)
```

Comment:
Please doxy this function

Caution: this function is in the BIG_ENDIAN section which is some flavors of Linux and Mac